src/ocm/components/clusterWizard/ClusterWizardContextProvider.tsx

```diff
@@ -14,15 +13,59 @@ import { StaticIpView } from '../clusterConfiguration/staticIp/data/dataTypes';
 import { getStaticIpInfo } from '../clusterConfiguration/staticIp/data/fromInfraEnv';
 import { Cluster, InfraEnv, AssistedInstallerOCMPermissionTypesListType } from '../../../common';
 import useSetClusterPermissions from '../../hooks/useSetClusterPermissions';
+import useClusterCustomManifests from '../../hooks/useClusterCustomManifests';
+import { ListManifestsExtended } from '../clusterConfiguration/manifestsConfiguration/data/dataTypes';
 
-const getWizardStepIds = (staticIpView?: StaticIpView): ClusterWizardStepsType[] => {
-  const stepIds: ClusterWizardStepsType[] = [...defaultWizardSteps];
+const addStepToClusterWizard = (
```

Comment:
For the "add", instead of passing the position, I would pass it the step after which it must be added.

Again, if new steps are introduced, we won't need to update the positions.

So you could do something like:
```
const addStepToClusterWizard = (
  wizardStepIds: ClusterWizardStepsType[],
  addAfterStep: ClusterWizardStepsType,
  itemsToAdd: ClusterWizardStepsType[],
) => {
  const referencePosition = wizardStepIds.findIndex((step) => step === addAfterStep);
  if (referencePosition !== -1) {
    wizardStepIds.splice(referencePosition + 1, 0, ...itemsToAdd);
  }
};

```